piptools/utils.py

```diff
@@ -358,3 +363,41 @@ def get_compile_command(click_ctx: click.Context) -> str:
                     left_args.append(f"{option_long_name}={shlex.quote(str(val))}")
 
     return " ".join(["pip-compile", *sorted(left_args), *sorted(right_args)])
+
+
+def get_required_pip_specification() -> SpecifierSet:
+    """
+    Returns pip version specifier requested by current pip-tools installation.
+    """
+    project_dist = get_distribution("pip-tools")
+    requirement = next(
+        (r for r in project_dist.requires() if r.name == "pip"), None
+    )  # pragma: no cover
```

Comment:
```suggestion
    )  # pragma: no branch
```

There is a [no branch](https://coverage.readthedocs.io/en/coverage-4.3.3/branch.html#structurally-partial-branches) pragma.